### ISSUE TITLE
NVDA can't return to browse

### ISSUE THREAD

**Issue by drlippman** [mathjax/MathJax-a11y#195]
Being brand new to using a screenreader, maybe I'm just missing a keystroke, but I can't figure out how to get out of a math equation after exploring it with `Shift-Space`.  I'm currently using Chrome, but had the same experience in FireFox.  I did notice the documentation says it's "difficult to continue in browse mode after using Explorer", so maybe that's what I'm experiencing, but it seems more than difficult.

Pressing `Esc` un-highlights the equation, but seems to leave NVDA in focus mode, where the arrow keys scroll the page rather than changing what's read.  Pressing `NVDA-space` returns to browse mode, but it starts browsing character-by-character in the equation and won't allow movement outside the equation.  The only way I seem able to get out is to `Tab` to a non-math item which will return to browse mode.

**Comment by pkra** [comment]
Yes, that's a general problem. See https://github.com/mathjax/MathJax-a11y/tree/master/docs#browse-and-focus-mode

**Comment by drlippman** [comment]
Yeah, I saw that, and figured there probably wasn't a good solution.  I guess the other question I asked is the more relevant then - whether there's a way to prevent collapsing for shorter equations, which would decrease the frequency with which one is forced to use the Explorer.
